pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss

```
@@ -1167,6 +1165,16 @@ section.quote {
     }
   }
 
+  .tab-content-container {
+    padding: 30px;
+    border-radius: 0px 0px 8px 8px;
```

Nit: use `rem`s here
```suggestion
    padding: 1.875rem;
    border-radius: 0 0 0.5rem 0.5rem;
```